cpp/include/cudf/interop.hpp

```diff
@@ -391,8 +392,8 @@ std::unique_ptr<cudf::table> from_arrow(
 std::unique_ptr<cudf::column> from_arrow_column(
   ArrowSchema const* schema,
   ArrowArray const* input,
-  rmm::cuda_stream_view stream        = cudf::get_default_stream(),
-  rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
+  rmm::cuda_stream_view stream,
```

Comment:
We lost the default here (I'd add a suggestion but I'll probably screw up the spacing in the GUI).

Reply:
Good catch, thanks. Fixed.